test/LiquidityPool.tests.ts

```diff
@@ -371,6 +368,7 @@ describe("LiquidityPoolTests", function () {
             let toChainId = 1;
 
             await addTokenLiquidity(tokenAddress, liquidityToBeAdded, owner);
+            await executorManager.addExecutor(executor.address);
```

Comment:
Why is this needed in this function?

Reply:
For the sendFundsToUser call on line 371